scripts/infra/README.md

```diff
@@ -84,4 +84,35 @@ scripts/infra/cloud-instance.sh ec2 stop
 scripts/infra/cloud-instance.sh ec2 start
 # When you are done, you can delete the instance
 scripts/infra/cloud-instance.sh ec2 terminate
-```
\ No newline at end of file
+```
+
+### IBM Cloud Example
```

Comment:
This is going to be troublesome to maintain given it's mostly a dup of the above example.  Can we just point out any differences (other than ibm in the command) from the above example?

Reply:
Yup, can do :+1: